serve/middleware/routes/templates/elements/cem-serve-knob/cem-serve-knobs.js

```
@@ -0,0 +1,297 @@
+    switch (type) {
+      case 'boolean':
+        input = document.createElement('input');
+        input.type = 'checkbox';
+        // Don't set initial checked state - let sync handle it
+        input.checked = false;
+        break;
+
+      case 'enum':
+        input = document.createElement('select');
+        // Add empty option
+        const emptyOpt = document.createElement('option');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)